oas

```diff
@@ -98,27 +98,44 @@ paths:
                 
     
   /indices/download:  
```

Comment:
Would it not be better to include the field_urn and index in the url (/indices/{field_urn}/download/{index}

Also, the field_urn should also be a parameter of the search (ex: /indices/{field_urn}/search).